addons/mail/static/src/models/thread/thread.js

```diff
@@ -287,6 +287,9 @@ function factory(dependencies) {
                 }
                 return list;
             }, []);
+            if (channelIds.length === 0) {
+                return [];
```

Comment:
just `return;`

Reply:
Oh yes, I'm confused, I was convinced the method returned a list